src/main/java/frc/trigon/robot/utilities/Conversions.java

```diff
@@ -0,0 +1,16 @@
+package frc.trigon.robot.utilities;
+
+public class Conversions {
+    public static double hundredMsToSec(double ms)
+    {
+        return ms*10;
+    }
+    public static double hundredMsToMin(double ms)
+    {
+        return hundredMsToSec(ms)*60;
+    }
+    public static double hundredMsToTickPerMin(double ms)
+    {
+        return hundredMsToMin(ms)*2048;
+    }
```

Comment:
Why would you want tick to minute? tick per second is fine.... tick per minute is pretty meaningless